community-operators/descheduler/descheduler.v0.0.1.clusterserviceversion.yaml

```diff
@@ -4,38 +4,6 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: |-
```

Comment:
you don't want to keep the examples any longer?

Reply:
Ohh, this is part of revert commit. I didn't know that this section also holds examples. Thanks for identifying it. I will  keep this section as is then.